sdk/core/System.BinaryData/tests/BinaryDataTests.cs

```diff
@@ -1,17 +1,15 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-using System;
 using System.Collections.Generic;
 using System.IO;
 using System.Runtime.InteropServices;
 using System.Text;
 using System.Threading.Tasks;
-using Azure.Core.Serialization;
 using Moq;
```

Comment:
To my knowledge, we don't use Moq in dotnet/runtime.  Please avoid using it in the tests here.  Yeah, we'll have to convert NUnit to Xunit, but hopefully that's mechanical.